enigma/src/main/java/org/quiltmc/enigma/source/Decompiler.java

```diff
@@ -4,10 +4,5 @@
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 public interface Decompiler {
-	@Deprecated // use remapper specific one for easy doc inclusion
-	default Source getSource(String className) {
```

Comment:
Id keep this method, or rename it to `getUndocumentedSource`

Reply:
that's what I was thinking!